### PR TITLE
Add status tracking feature flag to terraform

### PR DESCRIPTION
**Pull request by azizova-leyla**
### Description

This could be set to true on staging.  Shouldn't be set in prod.

## Release notes:

Describe the change as it should be communicated to partners in the release notes.

### Checklist

- [ ] Added the correct label: < feature | bug | dependencies | infrastructure | ignore-for-release >
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)

Fixes #<issue_number>; Fixes #<issue_number>; Fixes #<issue_number>...

### CODE REVIEW

File: cloud/shared/variable_definitions.json [hunk 1]
```diff
@@ -97,5 +97,11 @@
     "secret": false,
     "tfvar": false,
     "type": "string"
+  },
+  "FEATURE_FLAG_STATUS_TRACKING_ENABLED": {
+    "required": false,
+    "secret": false,
+    "tfvar": false,
```
Comment (nbeloglazov) [inline cloud/shared/variable_definitions.json:104]: That should be true given that you pass it to the terraform.

File: cloud/shared/variable_definitions.json [hunk 2]
```diff
@@ -97,5 +97,11 @@
...
+    "type": "bool"
```
Comment (nbeloglazov) [inline cloud/shared/variable_definitions.json:105]: And apparently we don't support bool :/

**Review by nbeloglazov — CHANGES_REQUESTED** [review]

File: cloud/shared/variable_definitions.json (hunk 2 above)
Comment (azizova-leyla) [reply cloud/shared/variable_definitions.json:105]: we do now =)

File: cloud/shared/variable_definitions.json (hunk 1 above)
Comment (azizova-leyla) [reply cloud/shared/variable_definitions.json]: fixed.

**Review by nbeloglazov — APPROVED** [review]

**Pull request merged**